woocommerce/woocommerce-ios
# [CPP Onboarding] Implement onboarding loading UI

Part of #4611 

Implements a loading screen to be shown while the app is refreshing the onboarding state. Because the state is unlikely to change after a successful setup, I decided to only show a loading screen if the initial state was not `.completed`.

## To test

1. Go to Settings > In-Person Payments
2. If the store is correctly onboarded into payments, the loading screen shouldn't be visible, otherwise there should be a loading screen while the app refreshes the data.

Another good way to test this is to try it in a store that doesn't have WCPay installed, as that screen will have a "Refresh" button, which should show the loading screen again.

Light mode | Dark mode
-|-
![Simulator Screen Shot - iPhone 12 Pro - 2021-08-10 at 10 56 28](https://user-images.githubusercontent.com/8739/128840842-d0f0ca99-6d29-44b3-b790-ee36c0cc3649.png)|![Simulator Screen Shot - iPhone 12 Pro - 2021-08-10 at 11 12 17](https://user-images.githubusercontent.com/8739/128840847-aad854cc-1e8b-41ad-b165-1769a58473b2.png)


Update release notes:
